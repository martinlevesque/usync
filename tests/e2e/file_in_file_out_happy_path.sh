
set -e

go run main.go -in-file tests/e2e/fixtures/filein.txt -out-file /tmp/usynctest.txt
diff /tmp/usynctest.txt tests/e2e/fixtures/file_in_file_out_happy_path.expected