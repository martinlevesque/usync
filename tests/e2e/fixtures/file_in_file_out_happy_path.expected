1234
123
43